templates/typescript/api-single.mustache

```diff
@@ -0,0 +1,43 @@
+{{>licenseInfo}}
+
+import getJsonResponse from "../../helpers/getJsonResponse";
+import Service from "../../service";
+/* tslint:disable:no-unused-locals */
```

Comment:
Not from a template since it happens here: 

```
{{#imports}}
import { {{classname}} } from '../../typings/{{serviceName}}/models';
{{/imports}}
```

And Mustache does not support logic. I'm removing an unused model here: https://github.com/Adyen/adyen-node-api-library/blob/fd75bd4239fc8406bd48c96cdb8b76fd502fad02/Makefile#L51